app/enterprise/changelog.md

```diff
@@ -4,6 +4,127 @@ no_search: true
+  For Hybrid mode, all Control Planes and Data Planes belonging to the
+  same cluster returns the same cluster ID. For traditional database
+  type deployments, all Kong nodes pointing to the same database will
+  also return the same cluster ID.
+- Introduced a `kong.log.set_serialize_value`, which allows customizing
+  the output of `kong.log.serialize`.
+
+#### Plugins
+- The [HTTP Log](https://docs.konghq.com/hub/kong-inc/http-log/) (`http-log`) plugin 
+has been improved to allow you to add headers to the HTTP request, which will help you integrate 
+with many observability systems.
+- The [Key Authentication](https://docs.konghq.com/hub/kong-inc/key-auth/) (`key-auth`)
+plugin has two new configuration parameters: `key_in_header` and `key_in_query`. Both 
+are booleans and tell Kong whether to accept (true) or reject (false) passed in either 
+the header or the query string. Both default to “true.”
+- The [Request Size Limiting](https://docs.konghq.com/hub/kong-inc/request-size-limiting/) 
+(`request-size-limiting`) plugin has a new configuration `require_content_length` that 
+causes the plugin to ensure a valid Content-Length header exists before reading the request body.
+- The [Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/) 
+(`serverless-functions`) plugin introduces a sandboxing capability, is *enabled* by default, 
+and where only Kong PDK, OpenResty `ngx` APIs, and Lua standard libraries are allowed.
+
+#### Configuration
+- `client_max_body_size` and `client_body_buffer_size`, that previously
+  hardcoded to 10m, are now configurable through `nginx_admin_client_max_body_size` 
+  and `nginx_admin_client_body_buffer_size`.
+- Kong-generated SSL private keys now have `600` file system permission.
+- Properties `ssl_cert`, `ssl_cert_key`, `admin_ssl_cert`,
+  `admin_ssl_cert_key`, `status_ssl_cert`, and `status_ssl_cert_key`
+  is now an array: previously, only an RSA certificate was generated
+  by default; with this change, an ECDSA is also generated. On
+  intermediate and modern cipher suites, the ECDSA certificate is set
+  as the default fallback certificate; on old cipher suite, the RSA
+  certificate remains as the default. On custom certificates, the first
+  certificate specified in the array is used.
+- Kong now runs as a `kong` user if it exists; if user does not exist
+  in the system, the `nobody` user is used, as before.
+  
+### Dependencies
+- Bumped `kong-plugin-serverless-functions` from 1.0 to 2.1.
+- Bumped `lua-resty-dns-client` from 5.1.0 to 5.2.0.
+- Bumped `lua-resty-healthcheck` from 1.3.0 to 1.4.0.
+- Bumped `OpenSSL` from 1.1.1h to 1.1.1i.
+- Bumped `kong-plugin-zipkin` from 1.1 to 1.2.
+- Bumped `kong-plugin-request-transformer` from 1.2 to 1.3.
+
+### Fixes
+
+#### Core
+- Fixed issue where a Go plugin would fail to read `kong.ctx.shared` values set by LUA plugins.
+- Properly trigger `dao:delete_by:post` hook.
```

Comment:
What does this actually fix / why should a user care / when will a user notice a problem prior to this change?

Reply:
> What does this actually fix / why should a user care / when will a user notice a problem prior to this change?

@eskibars This question needs to be answered by the engineer who implemented it. @RobSerafini can you assist here or point to someone who can?

Reply:
Will fix this in GA.